torchci/components/TorchAgentPage/utils.tsx

```diff
@@ -246,7 +246,7 @@ export const renderMarkdownWithLinks = (
         ),
         th: ({ children, ...props }) => {
           // Filter out react-markdown specific props that conflict with HTML props
-          const { node, ...htmlProps } = props as any;
+          const { ...htmlProps } = props as any;
```

Comment:
This looks a bit sus I think as although the first variable isn't used, my common sense tells me that removing it shift everything left by one, which might not be what the latter logic expects? Not sure how to test this though